custom-tests.yaml

```
@@ -795,6 +795,22 @@ api:
       var instance = reusableInstances.audioContext.createGain();
   GamepadEvent:
     __base: var instance = new GamepadEvent('gamepadconnected');
+  Geolocation:
+    __base: var instance = navigator.geolocation;
+  GeolocationCoordinates:
+    __base: >-
+      <%api.GeolocationPosition:posCallback%>
+      function posCallback(pos) {
+        callback(pos.coords);
+      }
+  GeolocationPosition:
+    __base: >-
+      <%api.Geolocation:geo%>
+      geo.getCurrentPosition(function(pos) {
```

The reuse doesn't really help here I think, it's less code and easier to understand to inline:

```suggestion
      navigator.geolocation.getCurrentPosition(function(pos) {
```

---

It does simplify the code, but I wanted to also make sure that this test returns `false` if `navigator.geolocation` doesn't exist (if `Geolocation` isn't available, there's no way `GeolocationPosition` can be).